.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+
+
+# Pull Request Title
+Quickly summarize the purpose of your PR.
+
+## Related issues
+Give reviewers and interested parties a good idea of how this is related to other issues or pull requests.
+
+Example: `Related to #1`
+
+## Testing
+Describe how you plan to test, or how your changes have been tested.
```

Comment:
I would mention that you can run all the tests locally by running `./rust/run_all_checks` from repository's base directory.

Reply:
I'll add that, yeah. I've also edited the document with comments - feels safer when you don't have to worry about them accidentally posting the entire template in the PR. 😄